index.html

```diff
@@ -1,16 +1,26 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="philosophical_issues.html">Philosophical Issues</a></li>
+  <li><a href="clashes_between_ideas.html">Clashes between Ideas</a></li>
+
     <head>
```

Comment:
on line 11 and 12, include the <script.js> and the <style.css> files like this, and the cos:
```
<link rel="stylesheet" type="text/css" href="style.css">
<script src="script.js"></script>
```